File: hacker1/mario.c
#include <stdio.h>
#include <cs50.h>

void printSpace(int x);
void printHash(int y);

int main(void)
{
    
  int a;
  do
  {
    printf("The height of the pyramid:");
    a = GetInt();
  }
  while ((a > 23) || (a < 0));

  for (int i = 1; i <= a; i++)
  {
    printSpace(a-i);
    printHash(i);
    printf("  ");
    printHash(i);
    printf("\n");
  }

  return 0;

}

void printSpace(int x)
{

  for (int i = 0; i < x; i++)
    printf(" ");

}

void printHash(int y)
{

  for (int i = 0; i < y; i++)
    printf("#");
   
}

File: pset6/mario.py
#python implementation of mario pyramid

import cs50

def main():
    while True:
        print("The height of the pyramid:", end=" ")
        height = cs50.get_int()
        if height >= 0 and height < 23:
            break
        
    for i in range(height):
        print(" " * (height-i-1), end="") 
        print("#" *(i+1), end="") 
        print("  ", end="")
        print("#" *(i+1)) 

if __name__ == "__main__":
    main()
